homeassistant/components/netatmo/sensor.py

```diff
@@ -145,17 +157,9 @@ def _retry(_data):
                 module_items = config[CONF_MODULES]
                 for module_name in module_items:
                     if module_name not in data.get_module_names():
+                        _LOGGER.info("Module %s not found", module_name)
                         continue
-                    for condition in data.station_data.monitoredConditions(
-                            module_name):
-                        dev.append(
-                            NetatmoSensor(
-                                data,
-                                module_name,
-                                condition.lower(),
-                                data.station
-                            )
-                        )
+                    dev.extend(find_devices(data, [module_name]))
```

Comment:
You can make this simpler (avoid the loop) if you move the "if" into "find_devices()". Just a note, it's up to you which way you prefer.